### PR TITLE
Store a per-remote count incremented every time we rebuild it

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

File: lvfs/models.py [hunk 1]
```diff
@@ -1865,6 +1865,7 @@ class Remote(db.Model):
     name = Column(Text, nullable=False)
     is_public = Column(Boolean, default=False)
     is_dirty = Column(Boolean, default=False)
+    build_cnt = Column(Integer, default=0)
```
Comment (superm1) [inline lvfs/models.py:1868]: Should this perhaps be `BigInteger` to prevent a difficult to debug problem some day?

Comment (hughsie) [reply lvfs/models.py:1868]: Stable is currently on it's 306th rebuild, so i think we'll be fine for some time yet with Integer :)

Comment (superm1) [reply lvfs/models.py:1868]: But what about OEM embargo remotes?  Surely those are going up quicker?

Comment (hughsie) [reply lvfs/models.py:1868]: The largest value is 1386

Comment (superm1) [reply lvfs/models.py:1868]: OK, yeah overkill.  PR LGTM.

**Pull request merged**